### PR TITLE
beta release

### PR DESCRIPTION
**Pull request by marktoda**
bump

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "publishConfig": {
     "access": "public"
   },
-  "version": "1.6.0",
+  "version": "1.6.1-beta.1",
```
Comment (ewilz) [inline package.json:8]: not necessarily a patch, but I guess it doesn't matter how these beta versions are published?? will prob eventually be our ur 2.0

**Review by ewilz — APPROVED** [review]
with minor nit?? doesn't really matter..

File: package.json (hunk 1 above)
Comment (marktoda) [reply package.json:8]: ohh UR 2.0

Comment (marktoda) [reply package.json:8]: im just gonna send for now we can fix up later lol

Comment (ewilz) [reply package.json:8]: yan with 2.0 we can finally match up our npm versions with our UniversalRouter versions!

**Pull request merged**